src/libs/ReportUtils.ts

```diff
@@ -300,13 +299,12 @@ type OptimisticIOUReport = Pick<
     | 'ownerAccountID'
     | 'participantAccountIDs'
     | 'reportID'
-    | 'state'
     | 'stateNum'
+    | 'statusNum'
```

Comment:
NAB - I'm assuming this move was just to make it more consolidated? Since it was originally down there at L309

Reply:
Yep just for consistency